### ISSUE TITLE
Dedicated notify user not created after upgrade

### ISSUE THREAD

**Issue by djmattyg007** [openmediavault/openmediavault#1022]
**Describe the bug**

The `openmediavault-notify` user was not created automatically after upgrading the `openmediavault` package from `5.6.4` to `5.6.5`. I tried running `sudo dpkg-reconfigure openmediavault` too, but that didn't change anything.

**To Reproduce**

I upgraded an existing installation from `5.6.4` to `5.6.5`. No relevant user account was created.

**Expected behavior**

The `openmediavault-notify` user account should now exist.

**openmediavault Server (please complete the following information):**

- OS version: Linux hagrid.localdomain 5.10.0-0.bpo.5-amd64 #1 SMP Debian 5.10.24-1~bpo10+1 (2021-03-29) x86_64 GNU/Linux
- openmediavault version:

```
ii  omvextras-unionbackend           5.0.2                               all          union filesystems backend plugin for openmediavault
ii  openmediavault                   5.6.5-1                             all          openmediavault - The open network attached storage solution
ii  openmediavault-diskstats         5.1.0-1                             all          openmediavault disk monitoring plugin
ii  openmediavault-downloader        5.1                                 all          OpenMediaVault downloader plugin
ii  openmediavault-keyring           1.0                                 all          GnuPG archive keys of the OpenMediaVault archive
ii  openmediavault-omvextrasorg      5.6.1                               all          OMV-Extras.org Package Repositories for OpenMediaVault
ii  openmediavault-snapraid          5.0.7                               all          snapraid plugin for OpenMediaVault.
ii  openmediavault-symlinks          5.0                                 all          OpenMediaVault symlinks plugin
ii  openmediavault-unionfilesystems  5.1.2                               all          Union filesystems plugin for OpenMediaVault.
```

**Client (please complete the following information):**

SSH'd in with my terminal, running commands like `cat /etc/passwd` and `id openmediavault-notify`.

**Additional context**

The postfix `master.cf` was also not updated, so it still uses the `nobody` account.

**Comment by djmattyg007** [comment]
I upgraded to `openmediavault` version `5.6.5-2`, and the user still isn't present on my system. The postfix config file was also not updated.

Is this problem potentially being caused by me updating from a terminal rather than in the web GUI?

**Comment by votdev** [comment]
No, the problem is that the `postinst` code that creates the new user and marks the configuration as dirty is not triggered when the previous package version was `5.6.5-1`. It only works for versions <= `5.6.4`. I will fix the issue with the next version `5.6.6`.
